### ISSUE TITLE
release: Provide SBOMs as part of the Kata Containers release

### ISSUE THREAD

**Issue by fidencio** [kata-containers/kata-containers#6460]
For those not familiar with SBOMs, here's a good material to familiarise yourself with it: https://about.gitlab.com/blog/2022/10/25/the-ultimate-guide-to-sboms/

As part of Kata Containers release, we should start releasing SBOMs, preferably in the SPDX format, that consumers of our project can start to rely on those.

As the first step, we can generate a high-level SBOMs for the components we ship, and later on go in details on how to do that with the shipped rootfs image.

**Comment by c3d** [comment]
Good idea in general. However, I think it would be nice to decide who would use them and how. No point in doing extra work if we don't have a consumer to at least test it.

**Comment by riteshnoronha** [comment]
This sounds like a great idea, the very next question after sbom generation is how consumable they are. We have created an open-source tool to help u answer this question. 

https://github.com/interlynk-io/sbomqs

`sbomqs share <sbom-file>`
Sample Output: https://sbombenchmark.dev/user/score?id=eb4903f6-88df-46bd-adb1-e5ea85cdc88f
